client/src/pages/AssessmentPage/AssessmentPage.jsx

```diff
@@ -0,0 +1,183 @@
+import React from "react";
+import {
+  Typography,
+  CssBaseline,
+  Button,
+  TextField,
+  Box,
+  FormControl,
+  RadioGroup,
+  FormControlLabel,
+  Radio
+} from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+const useStyles = makeStyles(theme => ({
+  root: {
+    display: "flex",
+    flexWrap: "wrap"
+  },
+  question: {
+    width: "75vw",
+    maxWidth: "300px"
+  }
+}));
+
+//TODO: Import the question from the database and update the state values
+
+function AssessmentPage(props) {
+  const classes = useStyles();
+  const [value, setValue] = React.useState("");
```

Comment:
Currently when one MC answer is chosen for one question, the user's answer for the other questions change as well. I think a fix for this could be to make the _value_ state an object, mapping each question to its answer value. 

(nitpick) I'd use a different variable name than _value_ since that's not very clear in showing what this state is supposed to represent.